engine/Real.cs

```diff
@@ -302,9 +304,16 @@ public Real(Real r)
 		public static explicit operator long(Real real) => (long) real._v;
 		public static explicit operator ulong(Real real) => (ulong) real._v;
 
+		/// <summary>
+		/// Convert a Real to a Complex with zero imaginary part.
+		/// </summary>
+		/// <param name="real">Real to convert.</param>
+		/// <returns>Complex with real part equal to the provided Real (as double) and zero imaginary part.</returns>
+		public static explicit operator Complex(Real real) => new Complex((double) real._v, 0.0);
+
 		#endregion
 
-#region Interfaces
+		#region Interfaces
```

Comment:
```suggestion
#region Interfaces
```
(These are just to keep things style-consistent with the rest of the file.)

Reply:
Done. I kinda forgot the "commit suggestion" button exists, I should have just used it...